05LessonFive/Checkpoint1/css/style.css

```diff
@@ -1,43 +1,39 @@
-.lawn {
-    width: 100%;
-    height: 300px;
-}
+/*CSS for Berries*/
 
-.lawn h1 {
-    text-align: center;
-    font-family: monospace;
-    color: darkgray;
+.image {
+  width:500px;
+  float:left;
 }
 
-.lawn {
-    background-image: url(../img/banner-garden-stmichaelsmount.jpg);
-    background-repeat: no-repeat;
-    background-position: center;
-    background-size: cover;
+.image img {
+  width:100%;
+  height:100%;
```

Comment:
once you set the width or height equal to 100%, they opposite one will automatically be 100% to constrain those proportions. not wrong here, just redundant.